ftl/node/main.py

```diff
@@ -99,7 +105,8 @@ def main(args):
         # Create (or pull from cache) the base image with the
         # package descriptor installation overlaid.
         logging.info('Generating dependency layer...')
-        with bldr.CreatePackageBase(base_image, cash) as deps:
+        with bldr.CreatePackageBase(base_image, cash,
+                                    eval(args.cache)) as deps:
```

Comment:
Why do we need the "eval" here? Would a "bool" work?

Reply:
So the flag gets passed in a string, and the bool of a non-empty string will always be True, so I used eval instead!